aya/src/maps/array/program_array.rs

```diff
@@ -26,26 +26,28 @@ use crate::{
 ///
 /// # Examples
 /// ```no_run
-/// # let bpf = aya::Bpf::load(&[])?;
+/// # let mut bpf = aya::Bpf::load(&[])?;
 /// use aya::maps::ProgramArray;
-/// use aya::programs::{CgroupSkb, ProgramFd};
+/// use aya::programs::CgroupSkb;
 /// use std::convert::{TryFrom, TryInto};
 ///
 /// let mut prog_array = ProgramArray::try_from(bpf.map_mut("JUMP_TABLE")?)?;
-/// let prog_0: &CgroupSkb = bpf.program("example_prog_0").unwrap().try_into()?;
-/// let prog_1: &CgroupSkb = bpf.program("example_prog_1").unwrap().try_into()?;
-/// let prog_2: &CgroupSkb = bpf.program("example_prog_2").unwrap().try_into()?;
-///
+/// let prog_0: &mut CgroupSkb = bpf.program_mut("example_prog_0").unwrap().try_into()?;
+/// let prog_0_fd =  prog_0.fd().unwrap();
```

Comment:
Why did you make it program_mut? Shouldn't this now be `let prog_0 = bpf.program("example_prog_0").unwrap().fd().unwrap();`

Reply:
Because `fd` required `&mut self`. Fixed it to be `&self` so `program()` works as expected